resources/assets/sass/app.scss

```diff
@@ -1,2 +1,146 @@
-// @import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap";
+//Globel Styles
 
+h1 {
+ font-size: 3.75em;
+}
+
+h2 {
+ font-size: 3em;
+}
+
+h3 {
+ font-size: 2.25em;
+}
+
+h4 {
+ font-size: 1.875em;
+}
+
+h5 {
+ font-size: 1.5em;
+}
+
+h6{
+ font-size: 1.125em;
+}
+
+
+.pr-button {
+ width:80%;
+ padding: 5%;
+ background: #ff4b2e;
+ border-radius: 10px;
+ border-bottom: #fc907e 3px solid;
+ border-top:none;
+ border-right:none;
+ border-left: none;
+ margin: 10% 0;
+ flex: 1 1 auto;
+}
+
+.divider {
+ width: 90%;
+ height: 2px;
+ background: #acacac;
+}
+
+//Brand Bar
+.brand {
+ background: #ff4b2e;
+ height: 20%;
+}
+
+
+.brand h2 {
+ font-family:'Reklame Script', cursive ;
+ color: #ffffff;
+ text-align: center;
+ margin: 0;
+ padding: 5px;
+
+}
+
+//Bottom Navigation Bar
```

Comment:
How about we break this and Home Styling out into separate files? Perhaps create a `modules` folder